main.go

```diff
@@ -200,6 +198,59 @@ func getTrailheadID(w http.ResponseWriter, userAlias string) string {
 	return userAlias
 }
 
+func doTrailheadAuraCallout(apexAction string, pageURI string) trailhead.Data {
+	// If config has been retrieved, try aura call
+	if 0 != len(auraConfig.FwUid) {
+		var trailheadData = doTrailheadCallout(
+			`message={"actions":[` + apexAction + `]}` +
+				`&aura.context=` + trailhead.GetAuraContext(auraConfig) + `&aura.pageURI=` + pageURI + `&aura.token="`)
+		// If the response is nil, try getting the new fwuid before failing
+		if trailheadData.Actions != nil {
+			return trailheadData
+		}
+	}
+
+	// Get fwuid from profile app config
+	updateAuraProfileAppConfig()
+
+	// Make aura call
+	if 0 != len(auraConfig.FwUid) {
+		return doTrailheadCallout(
+			`message={"actions":[` + apexAction + `]}` +
+				`&aura.context=` + trailhead.GetAuraContext(auraConfig) + `&aura.pageURI=` + pageURI + `&aura.token="`)
+	}
+
+	return trailhead.Data{Actions: nil}
+}
+
+// Aquire Profile App Config
+func updateAuraProfileAppConfig() {
+	//return "axnV2upVY_ZFzdo18txAEw"
+	url := trailblazerProfileAppConfig
+	method := "GET"
+
+	client := &http.Client{}
+	req, err := http.NewRequest(method, url, nil)
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Can you switch this to `log.Println()` instead of `fmt` and remove the import? `log` is preferred as it adds timestamps to the logged message automatically, `fmt` does not, example:

```
2020/08/16 14:45:13 [GET] "/trailblazer/matruff/certifications" 2.765204924s
2020/08/16 14:45:19 [GET] "/trailblazer/matruff/badges" 1.401023577s
2020/08/16 14:48:16 [GET] "/trailblazer/matruff/badges/superbadge" 2.030988269s
2020/08/16 14:48:53 [GET] "/trailblazer/matruff" 1.980818238s
```

Reply:
Copy and paste "bug" from before your refactor. Easy change, good catch.